mlos_bench/mlos_bench/config/cli/azure-redis-bench.jsonc

```diff
@@ -0,0 +1,25 @@
+//
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+//
+// Run:
+// mlos_bench --config mlos_bench/mlos_bench/config/cli/azure-redis-bench.jsonc --globals experiment_RedisBench.jsonc --tunable_values tunable-values/tunable-values-example.jsonc
+{
+    "config_path": [
+        "mlos_bench/mlos_bench/config",
+        "mlos_bench/mlos_bench/config/experiments",
```

Comment:
Can we document somewhere how "experiments" differs from "root"?

Reply:
@motus this is the main thing I wanted help crafting text on for one of the config dir READMEs.

Reply:
yeah, I think I should add README files to each of those directories to clarify things. Will do it now

Reply:
wrote more about global configs in [`mlos_bench/config/README.md`](mlos_bench/mlos_bench/config/README.md)